### ISSUE TITLE
Translations too long making runtime assertions fail

### ISSUE THREAD

**Issue by jamezrin** [alexemanuelol/rustplusplus#206]
Hey, I discovered the following error while using the bot with the "es" (Spanish) language selected:

```
Unhandled Rejection: ,Error: Invalid string length: undefined
ExpectedConstraintError > s.string.lengthLessThanOrEqual
  Invalid string length

  Expected: expected.length <= 45

  Received:
  | 'El comando personalizado para el Interruptor Inteligente:'

    at Object.run (/app/node_modules/@sapphire/shapeshift/src/constraints/util/urlValidators.ts:29:59)
    at /app/node_modules/@sapphire/shapeshift/src/validators/BaseValidator.ts:96:2
    at Array.reduce (<anonymous>)
    at StringValidator.parse (/app/node_modules/@sapphire/shapeshift/src/validators/BaseValidator.ts:96:2)
    at TextInputBuilder.setLabel (/app/node_modules/@discordjs/builders/src/components/textInput/TextInput.ts:64:36)
    at Object.getTextInput (/app/src/discordTools/discordTextInputs.js:28:38)
    at Object.getSmartSwitchEditModal (/app/src/discordTools/discordModals.js:83:68)
    at module.exports (/app/src/handlers/buttonHandler.js:461:37)
    at Object.execute (/app/src/discordEvents/interactionCreate.js:45:49)
    at DiscordBot.<anonymous> (/app/src/structures/DiscordBot.js:97:56)
    at DiscordBot.emit (node:events:512:28)
    at DiscordBot.emit (node:domain:489:12)
    at InteractionCreateAction.handle (/app/node_modules/discord.js/src/client/actions/InteractionCreate.js:97:12)
    at Object.module.exports [as INTERACTION_CREATE] (/app/node_modules/discord.js/src/client/websocket/handlers/INTERACTION_CREATE.js:4:36)
    at WebSocketManager.handlePacket (/app/node_modules/discord.js/src/client/websocket/WebSocketManager.js:352:31)
    at WebSocketShard.onPacket (/app/node_modules/discord.js/src/client/websocket/WebSocketShard.js:494:22)
    at WebSocketShard.onMessage (/app/node_modules/discord.js/src/client/websocket/WebSocketShard.js:328:10)
    at callListener (/app/node_modules/ws/lib/event-target.js:290:14)
    at WebSocket.onMessage (/app/node_modules/ws/lib/event-target.js:209:9)
    at WebSocket.emit (node:events:512:28)
    at WebSocket.emit (node:domain:489:12)
    at Receiver.receiverOnMessage (/app/node_modules/ws/lib/websocket.js:1184:20)`
```

This error occurred while editing a smart switch, as you can see the discord module has some limitations for the length of strings. I am submitting a pull request with alternatives that work with that 45-character limit.

**Comment by jamezrin** [comment]
Possible conflicting strings:

```
Field groupAddSwitchIdLabel in tr.json exceeds 45 chars and is a label, 
         Value: Eklemek istediğiniz şalterin kimlik numarası: (45 chars)

Field customTimerEditCrateOilRigUnlockLabel in it.json exceeds 45 chars and is a label, 
         Value: Tempo di sblocco della Cassa Bloccata della Piattaforma Petrolifera (secondi): (78 chars)

Field smartSwitchEditCommandLabel in it.json exceeds 45 chars and is a label, 
         Value: Il comando personalizzato per l'Interruttore Intelligente: (58 chars)

Field customTimerEditCrateOilRigUnlockLabel in sv.json exceeds 45 chars and is a label, 
         Value: Oljeplattformens låsta låda upplåsningstid (sekunder): (54 chars)

Field smartSwitchEditCommandLabel in sv.json exceeds 45 chars and is a label, 
         Value: Det anpassade kommandot för den Smarta Strömbrytaren: (53 chars)

Field trackerAddPlayerIdLabel in sv.json exceeds 45 chars and is a label, 
         Value: SteamID:t för spelaren som skall läggas till: (45 chars)

Field customTimerEditCrateOilRigUnlockLabel in ru.json exceeds 45 chars and is a label, 
         Value: Время открытия ящика на Нефтяной Вышке (сек): (45 chars)

Field customTimerEditCrateOilRigUnlockLabel in es.json exceeds 45 chars and is a label, 
         Value: Tiempo de desbloqueo de la Caja Bloqueada de la Petrolífera (segundos): (71 chars)

Field smartSwitchEditCommandLabel in es.json exceeds 45 chars and is a label, 
         Value: El comando personalizado para el Interruptor Inteligente: (57 chars)

Field customTimerEditCrateOilRigUnlockLabel in de.json exceeds 45 chars and is a label, 
         Value: Ölbohrinsel gesp. Kiste - Entsperrzeit (Sek): (45 chars)

Field groupEditCommandLabel in de.json exceeds 45 chars and is a label, 
         Value: Der benutzerdefinierte Befehl für die Gruppe: (45 chars)
```

**Comment by alexemanuelol** [comment]
Solved in: 5703215724b8fe4a0e4e13d9e488275697f14d66
